### PR TITLE
Emit TrackStartEvent

### PR DESCRIPTION
**Pull request by duncte123**
This adds the TrackStartEvent to the emitted events

Json is:
```JSON
{
    "op": "event",
    "type": "TrackStartEvent",
    "guildId": ".....",
    "track": "...."
}
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,9 @@
 Each release usually includes various fixes and improvements.
 The most noteworthy of these, as well as any features and breaking changes, are listed here.
 
+## v3.2.2
```
Comment (freyacodes) [inline CHANGELOG.md]: Don't edit this file

**Review by freyacodes — CHANGES_REQUESTED** [review]
Needs documentation in IMPLEMENTATION.md

File: CHANGELOG.md (hunk 1 above)
Comment (duncte123) [reply CHANGELOG.md]: I'll revert it

**Comment by duncte123** [comment]
> Needs documentation in IMPLEMENTATION.md

Where would you like me to document this as I cannot find any documentation for the other events in there

**Comment by freyacodes** [comment]
It's given as a Java example in IMPLEMENTATION.md.

Not pretty, but it's there.

**Comment by duncte123** [comment]
> It's given as a Java example in IMPLEMENTATION.md.
> 
> Not pretty, but it's there.

Then this is already present, it is the first file in the changes tab.

**Comment by freyacodes** [comment]
My bad. I must have assumed I was looking at actual source code.

**Review by freyacodes — APPROVED** [review]

**Pull request merged**